ledger_test.go

```diff
@@ -104,6 +104,65 @@ func TestLedger_BroadcastNop(t *testing.T) {
 	}
 }
 
+func TestLedger_Ignore(t *testing.T) {
+	testnet := NewTestNetwork(t)
+	defer testnet.Cleanup()
+
+	alice := testnet.faucet
+	bob := testnet.AddNodeWithWallet(t, "85e7450f7cf0d9cd1d1d7bf4169c2f364eea4ba833a7280e0f931a1d92fd92c2696937c2c8df35dba0169de72990b80761e51dd9e2411fa1fce147f68ade830a")
+	charlie := testnet.AddNodeWithWallet(t, "5b9fcd2d6f8e34f4aa472e0c3099fefd25f0ceab9e908196b1dda63e55349d22f03bb6f98c4dfd31f3d448c7ec79fa3eaa92250112ada43471812f4b1ace6467")
+
+	go func() {
+		timeout := time.NewTimer(time.Second * 10)
+		for {
+			select {
+			case <-timeout.C:
+				return
+
+			default:
+				_, err := alice.PlaceStake(1)
+				if err != nil {
+					t.Fatal(err)
+				}
+			}
+		}
+	}()
+
+	go func() {
+		for {
+			ri := <-charlie.WaitForRound(5)
+			if ri >= 5 {
+				break
+			}
+		}
+
+		fmt.Println("charlie: ps 100")
+		_, err := charlie.PlaceStake(100)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	for {
+		ri := <-bob.WaitForConsensus()
+		if ri == 0 {
+			return
+		}
+
+		<-alice.WaitForRound(ri)
+		<-charlie.WaitForRound(ri)
+
+		round := bob.ledger.Rounds().Latest()
+		fmt.Printf("round: %d, applied: %d, rejected: %d, ignored: %d, alice stake: %d, charlie stake: %d\n",
```

Comment:
Best not print any logs out in amidst a test (can leave it commented out), or otherwise use t.Logf.